lib/Config.js

```diff
@@ -968,6 +968,29 @@ class Config extends EventEmitter {
             this.replicationGroupId = 'RG001';
         }
 
+        if (process.env.HOSTNAME) {
+            // If running in Kubernetes, the pod name is set in the HOSTNAME env var
+            // and is of the form: cloudserver-7869f99955-nr984
+            // where:
+            // - cloudserver is the name of the pod
+            // - 7869f99955 is a long hash that guarantees uniqueness within the namespace
+            // - nr984 is a short hash that is unique only within the current ReplicaSet/Deployment
+            // We use the last part of the pod name as the instanceId, prefixed with 'i' for internal
+            // or 'e' for external cloudserver.
+            // This allows having a unique identifier that will be used when generating versionIds
+            const instanceType = process.env.HOSTNAME?.includes('internal') ? 'i' : 'e';
```

Comment:
The `?` is not needed as we already check above

```suggestion
            const instanceType = process.env.HOSTNAME.includes('internal') ? 'i' : 'e';
```